http_service/bugbug_http/download_models.py

```diff
@@ -4,14 +4,14 @@
 # You can obtain one at http://mozilla.org/MPL/2.0/.
 
 from bugbug import db, repository, test_scheduling, utils
-from bugbug_http.models import MODELS_NAMES, get_model
+from bugbug_http.models import MODEL_CACHE, MODELS_NAMES
 
 
 def download_models():
     for model_name in MODELS_NAMES:
         utils.download_model(model_name)
         # Try loading the model
-        get_model(model_name)
+        MODEL_CACHE.get(model_name)
```

Comment:
Here we have to allow missing models like you were doing before in check_models.py